### PR TITLE
Feature 33 countdown form

### PR DESCRIPTION
**Pull request by shmato4ek**

### CODE REVIEW

File: RecodoDesktop/RecodoDesktop/App.xaml.cs [hunk 1]
```diff
@@ -13,5 +7,9 @@ namespace RecodoDesktop
     /// </summary>
     public partial class App : Application
     {
+        void App_Startup(object sender, StartupEventArgs e)
+        {
+
+        }
```
Comment (Anatolii-Ivanov) [inline RecodoDesktop/RecodoDesktop/App.xaml.cs]: I don't think you need it now

File: RecodoDesktop/RecodoDesktop/Countdown.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                    {
+                        _timer.Stop();
+                        Hide();
+                    }
+                    _time = _time.Add(TimeSpan.FromSeconds(-1));                    
+                }, Application.Current.Dispatcher);
+			_timer.Start();
+		}
+	}
+}
```
Comment (Anatolii-Ivanov) [inline RecodoDesktop/RecodoDesktop/Countdown.xaml.cs]: check tabs

Comment (Anatolii-Ivanov) [reply RecodoDesktop/RecodoDesktop/Countdown.xaml.cs]: Add an ability to add constructor parameter function, which will be executed after countdown is finished

Comment (Anatolii-Ivanov) [reply RecodoDesktop/RecodoDesktop/Countdown.xaml.cs]: Move it to a function, minimum logic in a constructor

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]
Why do you have all that files? StickPanel, Icons, fonts...

**Review by Jyuart — CHANGES_REQUESTED** [review]
Good start, but please update it with following features:
- It should be displayed full screen (with background blurred or something like this, so that the user do not miss the moment recording starts)
- Remove that red circle. It's too acid
- And the window control should not be visible (not the debug buttons)

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by Jyuart — APPROVED** [review]

**Pull request merged**